@navikt/core/css/darkside/timeline.darkside.css

```diff
@@ -367,9 +392,9 @@
   position: absolute;
```

Comment:
We have removed the "arrow" from Popover, Tooltip and Helptext. I think it would make sense to not include arrow in the new system to.

Reply:
I guess it would be nice to also remove the tsx elements associated with the arrows too:
- https://github.com/navikt/aksel/blob/main/@navikt/core/react/src/timeline/period/ClickablePeriod.tsx#L167-L175
- https://github.com/navikt/aksel/blob/main/@navikt/core/react/src/timeline/Pin.tsx#L138-L147

But would break the old solution, so should be done at a later time. Is it covered by the tasklist item in https://github.com/navikt/team-aksel/issues/657 for "Arrow-prop fjernes fra Popover" as is? I feel like it it could be renamed to cover _all_ arrow / popover components too? I could also add a todo with a link to that issue 🤔

Reply:
[Can just do as we did in Tooltip](https://github.com/navikt/aksel/blob/main/%40navikt/core/react/src/tooltip/Tooltip.tsx)

If themeContext -> dont render arrow